src/TableView.js

```diff
@@ -344,8 +344,6 @@ class TableView extends React.Component {
           sections={this.state.sections}
           additionalItems={this.state.additionalItems}
           tableViewStyle={this.props.tableViewStyle}
-          tableViewCellStyle={this.props.tableViewCellStyle}
```

Comment:
I think instead of removing it from the TableView, instead, we can do it like how CSS cascades. 
Set the TableView `tableViewCellStyle` and then apply the `cellStyle` from the Item if it has one.